extensions/amp-video/amp-video.md

```diff
@@ -95,6 +96,10 @@ This attribute is similar to the `controls` attribute in the HTML5 `video`. If t
 
 If present, the video will automatically loop back to the start upon reaching the end.
 
+##### crossorigin
+
+Required if a resource, like a `track` tag, is hosted on a different origin than document.
```

Comment:
technically doesn't apply to `sources` and `src` resources, let's just say `required if a `track` resource is ..` can't think of anything other than `track` that may need it.